tests/test_runner/test_hooks.py

```diff
@@ -355,6 +356,101 @@ def test_cosine_runner_hook(multi_optimziers):
     hook.writer.add_scalars.assert_has_calls(calls, any_order=True)
 
 
+@pytest.mark.parametrize('multi_optimziers', (True, False))
+def test_flat_cosine_runner_hook(multi_optimziers):
+    """xdoctest -m tests/test_hooks.py test_flat_cosine_runner_hook."""
+    sys.modules['pavi'] = MagicMock()
+    loader = DataLoader(torch.ones((10, 2)))
+    runner = _build_demo_runner(multi_optimziers=multi_optimziers)
+
+    with pytest.raises(ValueError):
+        # expected float between 0 and 1
+        FlatCosineAnnealingLrUpdaterHook(start_pct=-0.1)
+
+    # add momentum scheduler
+
+    hook_cfg = dict(
```

Comment:
If `FlatCosineAnnealingLrUpdaterHook` has no corresponding `Momentum Update Hook`, the hook_cfg can be removed.

Reply:
OK.